python/lsst/sconsUtils/utils.py

```diff
@@ -136,6 +136,7 @@ def libraryLoaderEnvironment():
 #  @brief Safe wrapper for running external programs, reading stdout, and sanitizing error messages.
 #
 #  Note that the entire program output is returned, not just a single line.
+# Returns strings not bytes
```

Comment:
`sconsUtils` code explicitly does not use standard docstrings. I think it's because pydoc can't parse the files (because the files can never be imported unless scons is active). I will fix the space problem though.